src/walletSdk/Customer/index.ts

```diff
@@ -0,0 +1,96 @@
+import { AxiosInstance } from "axios";
+import { Sep9InfoRequiredError, CustomerNotFoundError } from "../Exceptions";
+import {
+  CustomerInfoMap,
+  Sep12Status,
+  Sep12Type,
+  Field,
+  ProvidedField,
+  GetCustomerResponse,
+  AddCustomerResponse,
+} from "../Types";
+
+export class Sep12 {
+  private token;
+  private baseUrl;
+  private httpClient;
+  private headers;
+
+  constructor(token: string, baseUrl: string, httpClient: AxiosInstance) {
+    this.token = token;
+    this.baseUrl = baseUrl;
+    this.httpClient = httpClient;
+    this.headers = {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${this.token}`,
+    };
+  }
+
+  async getByIdAndType(id: string, type: string): Promise<GetCustomerResponse> {
+    const resp = await this.httpClient.get(
+      `${this.baseUrl}/customer?id=${id}&type=${type}`,
+      {
+        headers: this.headers,
+      },
+    );
+
+    if (!resp.data.id) {
+      throw new CustomerNotFoundError(id);
+    }
+    return resp;
+  }
+
+  async add(
```

Comment:
@acharb @Ifropc I am missing a way of properly handling `binary` files. According to the SEP, we could either upload it through [PUT /customer](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#request-1) as the last fields. Or use the new [POST /customer/files](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#post-request) endpoint. In both cases I believe the [Content-Type](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#content-type) header should be set to `multipart/form-data` which I am not finding in the current TS implementation (didn't check the Kotlin one)

Reply:
It's not supported in KT at the moment. I think we should support it eventually in TS

Reply:
good point. Adjusted it here wdyt?
https://github.com/stellar/typescript-wallet-sdk/pull/71/commits/85e5213c54d74270beb0b64b93e4474e81f9ace1